src/batou_scm/buildout.py

```diff
@@ -65,7 +65,7 @@ def verify(self):
                 self.__update_needed = True
             else:
                 self.__update_needed = False
-            for clone in self.source.clones.values():
+            for clone in list(self.source.clones.values()):
```

Comment:
```suggestion
            for clone in self.source.clones.values():
```

`list()` is not needed here.

Reply:
yeah. 2to3ism